modules/ROOT/pages/migration-core.adoc

```diff
@@ -94,18 +93,18 @@ The table below maps Mule 3.x components to their Mule 4 equivalents.
 | singleton-object | Use Java module or Spring module.
 | spring-object | Use Java module or Spring module.
 | synchronous-processing-strategy | The behavior related to flow components execution is the same as flows in 4.x but it doesn't always execute in the same thread as in 3.x.
-| transactional scope | Replaced with “try” scope.
-| username-password-filter | Removed.
-| validator | Removed.
+| transactional scope | The try scope replaced transactional scope.
```

Comment:
I don't know if scopes are supposed to be tagged with the tick marks. Like `try` scope and `transactional` scope.